src/layout/section/cardsSection/CardsSection.tsx

```diff
@@ -1,20 +1,34 @@
 import React from 'react'
 import cc from 'classcat'
 
+import { SectionContentSize } from '../../../layout/section/baseSection'
 import { QrCardProps } from '../../../qrCard'
 import { TripCardProps } from '../../../tripCard'
 import { StyledCardsSection } from './CardsSection.style'
 
 type CardsProps = TripCardProps | QrCardProps
+type ChildrenProps = React.ReactElement<CardsProps>[] | React.ReactElement<CardsProps>
 
 export type CardsSectionProps = Readonly<{
-  children: React.ReactElement<CardsProps>[] | React.ReactElement<CardsProps>
+  children: ChildrenProps
   className?: string
-  width?: string
+  contentSize?: SectionContentSize
 }>
 
-export const CardsSection = ({ children, className = '' }: CardsSectionProps) => (
-  <StyledCardsSection className={cc(['kirk-cardsSection-wrapper', className])}>
+export const CardsSection = ({
+  children,
+  className = '',
+  contentSize = SectionContentSize.SMALL,
+}: CardsSectionProps) => (
+  <StyledCardsSection
+    className={cc([
+      'kirk-cardsSection-wrapper',
+      {
+        'kirk-cardsSection-wrapper--large': contentSize === SectionContentSize.LARGE,
```

Comment:
should not be part of the API since we are able to retrieve that with CSS

Reply:
I'm not sure I understand how we can retrieve it with CSS? It's not the breakpoint value